tests/CMakeLists.txt

```diff
@@ -222,3 +222,18 @@ file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/local-file.toml
 "[types]
 Oz = \"F\"
 ")
+
+# Code linting with Python scripts
+find_package(PythonInterp REQUIRED)
```

Comment:
This is deprecated [here](https://cmake.org/cmake/help/latest/module/FindPythonInterp.html).

Reply:
Since CMake 3.12, but we support cmake version all the way back to 2.8.12 (which is the one available natively on CentOS 7 and other old distro used in HPC centers). I initially used `find_package(Python REQUIRED)`, but this failed on Travis. 

I could add a cmake version check around this though.

Reply:
That would work.